### ISSUE TITLE
Cannot Read Translations

### ISSUE THREAD

**Issue by ZandarKoad** [tomslominski/yourls-redirect-index#3]
After installing the plugin, moving  index.php, and activating it the plugin, I get this message:

> Notice: Cannot read file user/plugins/redirect-index/translations/redirindex-.mo. Make sure there is a language file installed. More info: http://yourls.org/translations in /home/-SERVERNAME-/-FOLDERNAME-/includes/functions-l10n.php on line 458

I went ahead and entered in a redirect URL, and got this message:

> Notice: Cannot read file user/plugins/redirect-index/translations/redirindex-.mo. Make sure there is a language file installed. More info: http://yourls.org/translations in /home/-SERVERNAME-/-FOLDERNAME-/includes/functions-l10n.php on line 458

> Warning: Cannot modify header information - headers already sent by (output started at /home/-SERVERNAME-/-FOLDERNAME-/includes/functions-l10n.php:458) in /home/-SERVERNAME-/-FOLDERNAME-/index.php on line 17

I'll slowly work on resolving these issues myself, just wanted to let others know in case they run into the same thing.

**Comment by ZandarKoad** [comment]
Looks like the creator of this plugin already brought up this same problem long ago:  https://github.com/YOURLS/YOURLS/issues/2009

But it's not clear to me what the solution is / was.  Still investigating.

**Comment by ZandarKoad** [comment]
The error mentions /includes/functions-l10n.php, but no such file exists.  So that's as far as I got.  :)

**Comment by ZandarKoad** [comment]
Okay, I found that file (straight up user error there).  It was right were it should be.  I downloaded the file, manually implemented the changes [shown here](https://github.com/YOURLS/YOURLS/commit/fc0580cc6397cf76f1905ad8424d77d0b09ad37f), replaced the file, and poof:  **everything works perfectly.**

It may be that I downloaded an older file set...  The download URL I used was the one linked to on the main YOURLS page:  https://codeload.github.com/YOURLS/YOURLS/zip/1.7.1

In that archive, the functions-l10n.php file is not up to date.

I highly suggest anyone who wants to install YOURLS navigate instead to https://github.com/YOURLS/YOURLS, and click on the green "Clone or Download" button to the right.  That will have the most up to date files.
